.travis.yml

```diff
@@ -179,6 +179,19 @@ matrix:
         - bash redox/uncomment-cargo.sh
         - redoxer build --verbose
         - bash redox/comment-cargo.sh
+      deploy:
+        provider: script
+        script:
+          - openssl aes-256-cbc -K $encrypted_441570148e04_key -iv
+            $encrypted_441570148e04_iv -in redox/deploy_key.pem.enc -out
```

Comment:
I'm not sure about having a deployment key in the repo with access to the repository. What is the purpose of this deploy action? To merge the master branch into the redox-release branch? It might be good enough to once in a while check the master branch on redox compatibility.

The redox package could then refer to a specific commit that worked, or even better a tag to a released version.

Reply:
Btw, it might be nice to link to the redox package from the readme file, so redox users can install rustpython.

Reply:
Yeah, I suppose I could merge master to it manually every so often; it really wouldn't be that much of a hassle. I'll remove the automatic deployment and just leave the `BRANCH=redox-release`.